addons/diagnostics/common/src/main/java/org/commonjava/indy/diag/data/DiagnosticsManager.java

```diff
@@ -139,4 +143,49 @@ public File getDiagnosticBundle()
 
         return out;
     }
+
+    public File getRepoBundle() throws IOException
+    {
+        File out = createTempFile( "repos" );
+        logger.info( "Writing repo bundle to: '{}'", out );
+
+        try (ZipOutputStream zip = new ZipOutputStream( new FileOutputStream( out ) ))
+        {
+            zipRepositoryFiles( zip );
+        }
+        return out;
+    }
+
+    private void zipRepositoryFiles( ZipOutputStream zip ) throws IOException
+    {
+        DataFile[] packageDirs = dataFileManager.getDataFile( INDY_STORE ).listFiles( ( f ) -> true );
```

Comment:
I wonder if it would make sense to dump the repo definitions as they exist in the StoreDataManager instead. Currently, those are the same thing, but when we move to a cluster-enabled Indy implementation we're going to need to escape the filesystem for things like repo definition storage, and use an ISPN cache or similar instead.

Reply:
I add a TODO here. Let's use this for now until we move to new storage. I reckon this is a low priority.